node.gyp

```diff
@@ -666,5 +667,60 @@
         'test/cctest/util.cc',
       ],
     }
-  ] # end targets
+  ], # end targets
+
+  'conditions': [
+    ['OS=="aix"', {
+      'targets': [
+        {
+          'target_name': 'node',
+          'type': 'executable',
+          'dependencies': ['<(node_core_target_name)', 'node_exp'],
+
+          'include_dirs': [
+            'src',
+            'deps/v8/include',
+          ],
+
+          'sources': [
+            'src/node_main.cc',
+            '<@(library_files)',
+            # node.gyp is added to the project by default.
+            'common.gypi',
+          ],
+
+          'ldflags': ['-Wl,-bbigtoc,-bE:<(PRODUCT_DIR)/node.exp'],
+          'actions': [
+            {
+              'action_name': 'node_binary_creation',
+              'inputs': ['<@(library_files)'],
+              'outputs': ['nothing'],
+              'action': ['echo', 'Linking final node binary']
+            }
+          ]
+        },
+        {
+          'target_name': 'node_exp',
+          'type': 'none',
+          'dependencies': [
+            '<(node_core_target_name)',
+          ],
+          'actions': [
+            {
+              'action_name': 'expfile',
+              'inputs': [
+                '<(OBJ_DIR)'
+              ],
+              'outputs': [
+                'node.exp'
+              ],
+              'action': [
+                'sh', 'tools/create_expfile.sh', '<@(_inputs)', '<(PRODUCT_DIR)/<@(_outputs)'
```

Comment:
Style: long line.

It's probably better to add the `<(PRODUCT_DIR)/` bit a few lines up because I don't think this would work if `'outputs'` had more than entry.